portal/v2/src/ClusterDetailListComponents/FeatureFlags.tsx

```diff
@@ -0,0 +1,177 @@
+import { IShimmerStyles, Shimmer, ShimmerElementType } from '@fluentui/react/lib/Shimmer';
+import { Component } from "react"
+import { Stack, Text, IStackStyles, IStackItemStyles } from '@fluentui/react';
+import { contentStackStylesNormal } from "../App"
+import { FeatureFlags } from "../ClusterDetailList"
+
+interface FeatureFlagsComponentProps {
+    item: any
+    clusterName: string
+}
+
+interface IFeatureFlagsComponentState {
+    item: FeatureFlags
+}
+
+export const ShimmerStyle: Partial<IShimmerStyles> = {
+    root: {
+        margin: "11px 0"
+    }
+}
+
+export const headShimmerStyle: Partial<IShimmerStyles> = {
+    root: {
+        margin: "15px 0"
+    }
+}
+
+export const headerShimmer = [
+    { type: ShimmerElementType.line, height: 32, width: '25%' },
+]
+
+export const rowShimmer = [
+    { type: ShimmerElementType.line, height: 18, width: '75%' },
+]
+
+export const KeyColumnStyle: Partial<IStackStyles> = {
+    root: {
+        paddingTop: 10,
+        paddingRight: 15,
+    }
+}
+
+export const ValueColumnStyle: Partial<IStackStyles> = {
+    root: {
+        paddingTop: 10,
+    }
+}
+
+export const KeyStyle: IStackItemStyles = {
+    root: {
+        fontStyle: "bold",
+        alignSelf: "flex-start",
+        fontVariantAlternates: "bold",
+        color: "grey",
+        paddingBottom: 10
+    }
+}
+
+export const ValueStyle: IStackItemStyles = {
+    root: {
+        paddingBottom: 10
+    }
+}
+
+const featureFlagsHeadings: FeatureFlags = {
+  "aro.alertwebhook.enabled": "ARO Alertwebhook Enabled",
+  "aro.autosizednodes.enable": "ARO Autosizednodes Enable",
+  "aro.azuresubnets.enabled": "ARO Azuresubnets Enabled",
+  "aro.azuresubnets.nsg.managed": "ARO Azuresubnets NSG Managed",
+  "aro.azuresubnets.serviceendpoint.managed": "ARO Azuresubnets Serviceendpoint Managed",
+  "aro.banner.enabled": "ARO Banner Enabled",
+  "aro.checker.enabled": "ARO Checker Enabled",
+  "aro.dnsmasq.enabled": "ARO Dnsmasq Enabled",
+  "aro.genevalogging.enabled": "ARO Genevalogging Enabled",
+  "aro.imageconfig.enabled": "ARO Imageconfig Enabled",
+  "aro.machine.enabled": "ARO Machine Enabled",
+  "aro.machinehealthcheck.enabled": "ARO Machinehealthcheck Enabled",
+  "aro.machinehealthcheck.managed": "ARO Machinehealthcheck Managed",
+  "aro.machineset.enabled": "ARO Machineset Enabled",
+  "aro.monitoring.enabled": "ARO Monitoring Enabled",
+  "aro.nodedrainer.enabled": "ARO Nodedrainer Enabled",
+  "aro.pullsecret.enabled": "ARO Pullsecret Enabled",
+  "aro.pullsecret.managed": "ARO Pullsecret Managed",
+  "aro.rbac.enabled": "ARO RBAC Enabled",
+  "aro.routefix.enabled": "ARO Routefix Enabled",
+  "aro.storageaccounts.enabled": "ARO Storageaccounts Enabled",
+  "aro.workaround.enabled": "ARO Workaround Enabled",
+  "rh.srep.muo.enabled": "RH SREP MUO Enabled",
+  "rh.srep.muo.managed": "RH SREP MUO Managed",
+}
```

Comment:
We need to move away from renaming these flags. SRE's know and look for them as aro.alertwebhook.enabled, etc. and not by the names being used here.

Also, as Amber originally mentioned, we need to auto-populate any new feature flags without having to do an Admin Portal update for each one. For example, I edited my test cluster and added `aro.new.feature.enabled:  true` and of course it was not displayed. As we move over to using Liveconfig this adaptability is important.

Reply:
Fixed it, and kept the naming convention as `aro.new,feature.enabled: true`

Reply:
@Shivkumar13  you addressed the renaming of the feature flags, but I am still unable to see any new features that I add manually.

If you `oc edit cluster` and add a new feature into the list, it should come up in the list the next time you open the Feature Flags blade in the UI. Could you support that, so that we don't have to manage a list of feature flags in the Admin Portal?